config/release/kubemacpool.yaml

```diff
@@ -43,7 +45,10 @@ webhooks:
     - CREATE
     resources:
     - pods
-- clientConfig:
+  sideEffects: None
+- admissionReviewVersions:
```

Comment:
similarly, this needs to hold the **vm**'s supported version(since this webhook checks vm), right?